buildmodel/commands.go

```diff
@@ -340,9 +353,13 @@ func SubmitUpdatePR(f *PRFlags) error {
 		skipReason = "Dry run"
 	case *f.origin == "":
 		skipReason = "No origin specified"
-	case *f.githubPAT == "":
-		skipReason = "github-pat not provided"
-	case *f.githubPATReviewer == "":
+	case *f.gitHubPAT == "" && *f.GitHubAppClientID == "":
+		skipReason = "github-pat and github-app-client-id not provided"
+	case *f.GitHubAppClientID != "" && *f.GitHubAppInstallation == 0:
+		skipReason = "github-app-installation not provided"
+	case *f.GitHubAppClientID != "" && *f.GitHubAppPrivateKey == "":
+		skipReason = "github-app-private-key not provided"
```

Comment:
This seems error prone, I'd expect the auther flags abstraction should be able to help.